src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getAllUsers() {
```

Comment:
we are using ts so it is better to specify the expected return value type for this function:
```suggestion
export async function getAllUsers(): Promise<User[]> {
```
here we expect to fetch and return users list, so we are using `User[]` as return type. And because async functions return new Promise, we are using Promise generic.